pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -19,33 +19,243 @@ package dockershim
 import (
 	"fmt"
 
+	dockertypes "github.com/docker/engine-api/types"
+	dockercontainer "github.com/docker/engine-api/types/container"
+	dockerfilters "github.com/docker/engine-api/types/filters"
+
 	runtimeApi "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
+const (
+	defaultSandboxImage = "gcr.io/google_containers/pause-amd64:3.0"
+
+	// Various default sandbox resources requests/limits.
```

Comment:
Do we already have these default values somewhere in our current code, or they are added in this PR? I couldn't find them in our code, maybe I missed something. :)